README.md

```diff
@@ -284,6 +284,22 @@ six.assertRaisesRegex(self, e, r, fn)   # self.assertRaisesRegex(e, r, fn)
 six.assertRegex(self, s, r)             # self.assertRegex(s, r)
 ```
 
+### `open` alias
+
+Availability:
+- `--py3-plus` is passed on the commandline.
+
+```python
+# input
+import io
+with io.open('f.txt') as f:
+    pass
+# output
+import io
```

Comment:
I would drop the imports in the example, just to minimize confusion

Reply:
done